src/Serilog/ReflectionTypesScalarDestructuringPolicy.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Reflection;
+using Serilog.Core;
+using Serilog.Events;
+
+namespace Serilog.Policies
+{
+    class ReflectionTypesScalarDestructuringPolicy : IDestructuringPolicy
+    {
+        public bool TryDestructure(object value, ILogEventPropertyValueFactory propertyValueFactory, out LogEventPropertyValue result)
+        {
+            var type = value as System.Type;
```

Comment:
Any reason not to keep the impl from above? Seems equivalent and a bit more terse:

```csharp
           if (value is Type || value is MemberInfo)
           {
               result = new ScalarValue(value);
               return true;
           }

            result = null;
            return false;
```